vendor/istio.io/api/mesh/v1alpha1/config.proto

```diff
@@ -306,8 +306,14 @@ message MeshConfig {
   // It can be enabled by destination using the destinationRule.trafficPolicy.connectionPool.http.h2UpgradePolicy override.
   H2UpgradePolicy h2_upgrade_policy = 41;
 
```

Comment:
pls update istio/api first (https://github.com/istio/api/blob/master/mesh/v1alpha1/config.proto), then update sha here; btw could we use env variable here ?(it's not recommended to update meshconfig /cc @costinm )

Reply:
>pls update istio/api first then update sha here

that's how I did it; I pulled in my changes using `GO111MODULE=on go mod vendor` after changing `go.mod`

>btw could we use env variable here ?

that's what I was not sure about. this version is changing the MeshConfig, I am working on a document outlining this plus another version that doesn't change the MeshConfig